### PR TITLE
Publish binary artifacts to GitHub

### LINKED ISSUE #457: Publish binary version of Authelia in Github artifacts

### PR DESCRIPTION
**Pull request by nightah**
You need to configure your `GITHUB_API_KEY` in [Travis](https://stackoverflow.com/questions/23277391/how-to-publish-to-github-pages-from-travis-ci).

Closes #457.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -50,6 +49,17 @@ jobs:
         - kill %1
       after_success:
         - authelia-scripts docker push-image --arch=$ARCH
+        - if [[ $TRAVIS_TAG ]]; then docker create --name authelia-binary clems4ever/authelia:$TRAVIS_TAG-$ARCH && docker cp authelia-binary:/usr/app/authelia ./authelia-linux-$ARCH; fi
```
Comment (clems4ever) [inline .travis.yml]: It's not gonna be sufficient, we need to tar.gz the binary along with the compiled frontend until we have a better solution for packaging the frontend.

File: .travis.yml [hunk 2]
```diff
@@ -50,6 +49,17 @@ jobs:
...
+      before_deploy:
+        - echo Deploying Authelia $TRAVIS_TAG-$ARCH to GitHub Releases
+      deploy:
+        provider: releases
+        api_key: '$GITHUB_API_KEY'
```
Comment (clems4ever) [inline .travis.yml]: I will create the API key and set it in Travis.

Comment (clems4ever) [reply .travis.yml]: Done

File: .travis.yml (hunk 1 above)
Comment (nightah) [reply .travis.yml]: You're absolutely right here, that's fine I'll grab the `public_html` folder and tar it up with the binary and adjust accordingly.

File: .travis.yml (hunk 2 above)
Comment (nightah) [reply .travis.yml]: Great.

**Pull request merged**